### ISSUE TITLE
Conditional assembly does not evade tag definitions

### ISSUE THREAD

**Issue by jamque** [Fubukimaru/asMSX#90]
Take this code
```
USE_SCREEN=5
  ZILOG
  BIOS
  ROM
  START    GAME
  
GAME:
  
IF USE_SCREEN == 4
  call SCREEN4RENDER
ENDIF

IF USE_SCREEN == 5
  call SCREEN5RENDER
ENDIF

SCREEN5RENDER:
  ret
```
and asmsx says in reference to call SCREEN4RENDER
```
TEST.ASM, line 10: undefined identifier
```
Shouldn't it ignore the tags inside the IF before looking to see if that CALL SCREEN4RENDER exists?

**Comment by Fubukimaru** [comment]
Related with #58

**Comment by duhow** [comment]
Fixed in #113
